packages/odyssey-react/src/components/List/List.theme.ts

```diff
@@ -0,0 +1,24 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import type { ThemeReducer } from "@okta/odyssey-react-theme";
+
+export const theme: ThemeReducer = (theme) => ({
+  GridColumnGapDescription: theme.SpaceRemM,
+  GridRowGapDescription: theme.SpaceRemXs,
+  MaxWidth: theme.FontLineLengthMax,
```

Comment:
I've been prefixing this with `Font` i.e. `FontMaxWidth` since it's organized under the same grouping for global tokens. Trying to aim for consistency even if we one day decide MaxWidth shouldn't be font related and decide to change everything all together!